creusot/src/resolve.rs

```diff
@@ -160,6 +160,11 @@ impl<'body, 'tcx> EagerResolver<'body, 'tcx> {
         self.dead_locals()
     }
 
+    pub fn frozen_locals_before(&mut self, loc: Location) -> BitSet<Local> {
```

Comment:
I'm not sure I'm a huge fan of the resolver exposing this api used for something other than resolving.

Reply:
Why do you think this is the appropriate place for this API?

Reply:
Ah, I forgot to reply, sorry. I don't think this necessarily is the right place for the API. It's just that this is where we compute the data-flow analysis, which I didn't want to duplicate.